### ISSUE TITLE
Still busy

### ISSUE THREAD

**Issue by ezar** [dolanmiu/MMM-awesome-alexa#7]
Only I can play one time... second time, system halt.

```
0|MagicMir | hotword 1 alexa
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy
```

**Comment by ezar** [comment]
0|MagicMir | hotword 1 alexa
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy
0|MagicMir | transiting to state: idle
0|MagicMir | hotword 1 alexa
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy
0|MagicMir | transiting to state: idle
0|MagicMir | hotword 1 alexa
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy
0|MagicMir | transiting to state: idle
0|MagicMir | hotword 1 alexa
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy

**Comment by dolanmiu** [comment]
Are your speakers working?

```
0|MagicMir | transiting to state: listening
0|MagicMir | transiting to state: busy
0|MagicMir | transiting to state: idle
```

This indicates to me that it did the full loop: `listening` `busy` (which is speaking to you), and `idle`, back the the beginning.

**Comment by ezar** [comment]
Yes it working. 
I ask to Alexa two or three times, after that hangs.
Regards,

**Comment by dolanmiu** [comment]
What are your OS and specs? For me, I can ask as many times as I want

**Comment by ezar** [comment]
Raspberry pi 3 using Raspbian Lite.

**Comment by dolanmiu** [comment]
I will try it on my Raspberry Pi over the weekend

I am running it on:

MacOS Sierra
Rasberry Pi 3 with Raspbian + PIXEL

**Comment by ezar** [comment]
@dolanmiu Do you tried?

**Comment by dolanmiu** [comment]
Yes, I have tried

Sorry about the delay

works fine for me, I tried it 6 times in a row on my Raspberry Pi, and it worked every single time

Just a quick check, on your Pi, can you run:

```
$ rec t.wav
```

If you can't, then there is something wrong with your mic

Raspberry Pi has weird issues with the mic:

https://www.raspberrypi.org/forums/viewtopic.php?t=13088&p=332703

**Comment by henrikra** [comment]
I have noticed the same on my mac that I can get alexa respond to me like 3 times. Then it stops responding 🤔

**Comment by dolanmiu** [comment]
@henrikra 

Found the issue:

https://github.com/electron/electron/issues/9201

**Comment by henrikra** [comment]
Hmm very interesting since when that error is in the log the Alexa won't hear me anymore. At electron they said it is not their fault. But MagicMirror is done with Electron so what was your idea that it can be MagicMirror bug? :D

**Comment by dolanmiu** [comment]
Because magic mirror uses Electron, and if they somehow don't utilize Electron properly, perhaps it messes up? just speculation :P

**Comment by henrikra** [comment]
This is fixed in #38
